src/ui-shell/header/header-menu.component.ts

```diff
@@ -25,17 +26,43 @@ import { Component, Input, HostBinding, HostListener } from "@angular/core";
 	`
 })
 export class HeaderMenu {
-	@Input() title;
+	@Input() title: string;
+	@Input() href = "javascript:void(0)";
```

Comment:
Can we a setter and getter like in `switcher-list-item.component.ts` instead of `getHref()`?

Reply:
Sure, that makes sense 👍